tests/flytekit/conftest.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+
+def pytest_sessionstart():
+    os.environ["FLYTE_SDK_USE_STRUCTURED_DATASET_OLD"] = os.environ.get("FLYTE_SDK_USE_STRUCTURED_DATASET_OLD", "FALSE")
```

Comment:
no need to have 2 fixtures, you can instead use `yield` to restore the old values, like so:
```
def pytest_session():
    old_value = os.environ["FLYTE_SDK_USE_STRUCTURED_DATASET"]
    # Use Structured Datasets. This should get picked up automatically by the loading of structured_dataset.py
    os.environ["FLYTE_SDK_USE_STRUCTURED_DATASET"] = "TRUE"
    yield
    os.environ["FLYTE_SDK_USE_STRUCTURED_DATASET"] = old_value